src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,4 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
it's better not to set `margin: 0` for all elements on page. just add this styles for specific selectors
```suggestion
```